.gitignore

```diff
@@ -24,3 +24,12 @@ bazel-*
 # IntelliJ
 .idea
 .ijwb
+
```

Comment:
Pending internal discussion on our `.gitignore` guidelines - will re-review once we've fleshed out the details!

Reply:
Let's keep typedb-studio and nuke the others.